src/js/_store/modules/categories.js

```diff
@@ -0,0 +1,32 @@
+import axios from '@Helpers/axiosDefault';
+
+export default {
+  namespaced: true,
+  state: {
+    loading: false,
+    errorMessage: '',
+    categoryList: [],
+  },
+  mutations: {
+    doneGetAllCategories(state, categories) {
+      state.categoryList = categories.reverse();
+    },
+    failRequest(state, { message }) {
+      state.errorMessage = message;
+      state.loading = false;
+    },
+  },
+  actions: {
+    getAllCategories({ commit, rootGetters }) {
+      axios(rootGetters['auth/token'])({
+        method: 'GET',
+        url: '/category',
+      }).then(({ data }) => {
+        const categoryList = data.categories;
+        commit('doneGetAllCategories', categoryList);
```

Comment:
[nits]

一時変数は逆に可読性を落とします。
1度しか使わないのであれば直接指定した方が読みやすいかなと思います。

```suggestion
        commit('doneGetAllCategories', data.categories);
```

Reply:
categoryListを定義して代入せず、直接してしました。